tests/integration/targets/azure_ops_test_azure_manage_snapshot/playbooks/main.yml

```diff
@@ -0,0 +1,59 @@
+---
```

Comment:
Other tests are run from tasks/main.yml. Not sure that the current structure can be run with the existing Jenkins job.

Reply:
We should be launching the tests the same way as ansible upstream does.  

```
ansible-test integration azure_ops_test_azure_manage_snapshot
```

Why is jenkins doing it differently?